apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -547,11 +548,20 @@ else
     || test_exit_code=$?
 fi
 
+# Run a post-action binary, if provided.
 post_action_binary=%(post_action_binary)s
-TEST_EXIT_CODE=$test_exit_code \
-  TEST_LOG_FILE="$testlog" \
-  SIMULATOR_UDID="$simulator_id" \
-  "$post_action_binary"
+if [[ -n "${result_bundle_path:-}" ]]; then
+  TEST_EXIT_CODE=$test_exit_code \
+    TEST_LOG_FILE="$testlog" \
+    SIMULATOR_UDID="$simulator_id" \
+    TEST_XCRESULT_BUNDLE_PATH="$result_bundle_path" \
```

Comment:
the user could theoretically delete this path in their post-action binary, and then encounter an error when it's deleted below. not the only bug of this kind that exists due to the pre-/post-action binaries (e.g. user could delete a simulator in the pre-action prior to testing)